ipapython/install/cli.py

```diff
@@ -58,7 +58,24 @@ def _get_usage(configurable_class):
 
 def install_tool(configurable_class, command_name, log_file_name,
                  debug_option=False, verbose=False, console_format=None,
-                 use_private_ccache=True, uninstall_log_file_name=None):
+                 use_private_ccache=True, uninstall_log_file_name=None,
+                 ignore_return_codes=[]):
```

Comment:
Please don't use mutable data types in argument lists. If you need an iterable, use a tuple as default argument ``ignore_return_codes=()``.